chain/actors/actor_miner.go

```diff
@@ -817,32 +767,34 @@ type DeclareFaultsParams struct {
 }
 
 func (sma StorageMinerActor) DeclareFaults(act *types.Actor, vmctx types.VMContext, params *DeclareFaultsParams) ([]byte, ActorError) {
-	oldstate, self, aerr := loadState(vmctx)
-	if aerr != nil {
-		return nil, aerr
-	}
+	/*
+		oldstate, self, aerr := loadState(vmctx)
```

Comment:
oh yeah... faults. Thats probably important to figure out, but maybe we can do it in a followup PR?

Reply:
I'd do this is a followup PR